advocacy_docs/partner_docs/VeritasGuide/05-UsingNetBackupForPostgreSQL.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: 'Using NetBackup for PostgreSQL'
+description: 'Walkthroughs of example usage scenarios'
+---
+
+This section shows how database backup/restore operations are performed for EDB Postgres Advanced Server using NetBackup for PostgreSQL.
+
```

Comment:
Think it would be helpful to have something like.  In this section we will show you how to do some of the most common backup/restore operations.   

bullet list of the operations with description of what they are.  Then go through them.